modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/home/components/ProjectsNavbar/components/ProjectCategoryDropdown/ProjectCategoryDropdown.jsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {Button, DropDown} from '@clayui/core';
+import ClayIcon from '@clayui/icon';
+import {useState} from 'react';
+import i18n from '~/common/I18n';
+import {Skeleton} from '~/common/components';
+import getKebabCase from '~/common/utils/getKebabCase';
+import {PROJECT_CATEGORY_GROUPS} from '~/routes/customer-portal/utils/constants/projectCategoryGroups';
+
+const ProjectCategoryDropdown = ({loading}) => {
+	const [active, setActive] = useState(false);
+	const [selectedItemIndex, setSelectedItemIndex] = useState(0);
+
+	const items = [
```

Comment:
```suggestion
	const [selectedProjectCategoryIndex, setSelectedProjectCategoryIndex] = useState(0);

	const projectCategoryItems = [
```

Please change to a more specific name, like projectCategoryItems.